stage1/init/kvm/hypervisor/hypervisor.go

```diff
@@ -44,4 +46,9 @@ func (hv *KvmHypervisor) InitKernelParams(isDebug bool) {
 			"quiet systemd.log_level=emerg",
 		}...)
 	}
+
+	customKernelParams := os.Getenv("RKT_HYPERVISOR_CUSTOM_KERNEL_PARAMS")
```

Comment:
I tried this:
```
$ sudo RKT_HYPERVISOR_CUSTOM_KERNEL_PARAMS="foo1=bar foo2=bar" ./build-rkt-1.21.0+git/target/bin/rkt run   --stage1-path=$PWD/build-rkt-1.21.0+git/target/bin/stage1-kvm.aci   --insecure-options=image --interactive   docker://busybox --cpu 1 --exec sh -- -c   'cat /proc/cmdline'
noapic noacpi pci=conf1 reboot=k panic=1 i8042.direct=1 i8042.dumbkbd=1 i8042.nopnp=1 console=ttyS0 i8042.noaux=1 rw rootflags=trans=virtio,version=9p2000.L,cache=mmap rootfstype=9p systemd.default_standard_error=journal+console systemd.default_standard_output=journal+console systemd.machine_id=61ebc2db2c28420a9d6402b168227ed6 console=hvc0 init=/usr/lib/systemd/systemd no_timer_check noreplace-smp tsc=reliable systemd.show_status=false systemd.log_target=null rd.udev.log-priority=3 quiet=vga quiet systemd.log_level=emerg foo1=bar foo2=bar
```
It seems to work as intended.

The string will be coalesced as a single space-embedding parameter anyway in [StartCmd](https://github.com/coreos/rkt/blob/master/stage1/init/kvm/hypervisor/hvlkvm/lkvm_driver.go#L53)